provision/acc_provision/acc_provision.py

```diff
@@ -958,6 +1132,27 @@ def Raise(exception):
             "net_config/pod_subnet": (get(("net_config", "pod_subnet")), required),
             "net_config/node_subnet": (get(("net_config", "node_subnet")), required),
         }
+    elif is_chained_mode(config):
+        checks = {
+            # ACI config
+            "aci_config/system_id": (get(("aci_config", "system_id")),
+                                     lambda x: required(x) and isname(x, 32)),
+            "aci_config/apic_refreshtime": (get(("aci_config", "apic_refreshtime")),
+                                            is_valid_refreshtime),
+            "aci_config/apic_refreshticker_adjust": (get(("aci_config", "apic_refreshticker_adjust")),
+                                                     is_valid_apic_refreshticker_adjust),
+            "aci_config/apic_subscription_delay": (get(("aci_config", "apic_subscription_delay")),
+                                                   is_valid_apic_sub_delay),
+            "aci_config/apic_host": (get(("aci_config", "apic_hosts")), required),
+            "aci_config/vrf/name": (get(("aci_config", "vrf", "name")), required),
+            "aci_config/vrf/tenant": (get(("aci_config", "vrf", "tenant")),
+                                      required),
+            "kube_config/image_pull_policy": (get(("kube_config", "image_pull_policy")),
+                                              is_valid_image_pull_policy),
+            # Network Config
+            "net_config/node_subnet": (get(("net_config", "node_subnet")),
```

Comment:
this is not required if skip_node_network_provisioning is set

Reply:
Resolved